public/global.css

```diff
@@ -1,63 +1,61 @@
-html, body {
-	position: relative;
-	width: 100%;
-	height: 100%;
+:root {
+  --font-regular: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+  --font-monospace: "Courier New", Courier, monospace;
 }
 
-body {
-	color: #333;
-	margin: 0;
-	padding: 8px;
-	box-sizing: border-box;
-	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
-}
-
-a {
-	color: rgb(0,100,200);
-	text-decoration: none;
+/* See https://www.30secondsofcode.org/css/s/box-sizing-reset */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
 }
 
-a:hover {
-	text-decoration: underline;
+html,
+body {
+  position: relative;
+  width: 100%;
+  height: 100%;
 }
 
-a:visited {
-	color: rgb(0,80,160);
+body {
+  margin: 0;
+  background-image: url("https://upload.wikimedia.org/wikipedia/commons/f/f3/43228_1_miscellaneous_digital_art_simple_simple_patterns.jpg");
+  background-size: cover;
+  font-family: var(--font-monospace);
+  color: white;
 }
 
-label {
-	display: block;
+button,
+input {
+  background-color: transparent;
+  color: inherit;
+  font-size: 1rem;
 }
 
-input, button, select, textarea {
-	font-family: inherit;
-	font-size: inherit;
-	-webkit-padding: 0.4em 0;
-	padding: 0.4em;
-	margin: 0 0 0.5em 0;
-	box-sizing: border-box;
-	border: 1px solid #ccc;
-	border-radius: 2px;
+input {
+  padding: 0.75rem;
```

Comment:
I'm using `rem` instead of `em` pretty much everywhere. The `em` unit can get you in a pickle. Check out [this article](https://www.digitalocean.com/community/tutorials/css-rem-vs-em-units) for more details.

Reply:
Good to know!